src/main/java/seedu/address/ui/ReaderViewStyle.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.ui;
+
+import static java.util.Objects.requireNonNull;
+
+import java.net.URL;
+import java.util.Locale;
+
+import seedu.address.MainApp;
+
+/**
+ * Available reader view styles.
+ */
+public enum ReaderViewStyle {
+    DEFAULT(requireNonNull(MainApp.class.getResource(Constants.STYLESHEETS_FILE_FOLDER + "default.css"))),
```

Comment:
I think ```ReaderViewStyle.class.getResource(...)``` works, then don't need to import ```MainApp```

Reply:
Thanks, didn't know this could be done!

Reply:
Fixed in 7ef2cc8